src/components/List.js

```diff
@@ -71,399 +71,82 @@ const List = (props) => {
-                      ""
-                    ) : (
-                      <li key={hosp[0]}>
-                        <span>
-                          <span className="bold">{hosp[2]}</span>:{" "}
-                          <span className="bold">
-                            <Separator number={hosp[1].inHospital.today} />
-                          </span>{" "}
-                          <Translate
-                            text={"oseba"}
-                            number={hosp[1].inHospital.today}
-                          ></Translate>{" "}
-                          (
-                          <span className="bold">
-                            +<Separator number={hosp[1].inHospital.in} /> -
-                            <Separator number={hosp[1].inHospital.out} />
-                          </span>
-                          ), EIT{" "}
-                          <span className="bold">
-                            <Separator number={hosp[1].icu.today} />
-                          </span>{" "}
-                          <Translate
-                            text={"oseba"}
-                            number={hosp[1].icu.today}
-                          ></Translate>{" "}
-                          (
-                          <span className="bold">
-                            +
-                            <Separator number={hosp[1].icu.in} /> -
-                            <Separator number={hosp[1].icu.out} />
-                          </span>
-                          ).
-                        </span>
-                        <br />
-                      </li>
-                    );
-                  })}
-          </ul>
-        </span>
-
-        <span className={check_third_mun}>
-          <p className="text">
-            <Arrow /> Po krajih:
-          </p>
-          <ul className="municipalities">
-            <Municipalities
-              data={municipalities}
-              showTrend="y"
-              icons="FB"
-            ></Municipalities>
-          </ul>
-          <Outro />
-          <br />
-          <br />
-          <p className="text">
-            <Arrow /> Po krajih:
-          </p>
-          <ul className="municipalities">
-            <Municipalities
-              data={municipalities}
-              showTrend="y"
-              icons="TW"
-            ></Municipalities>
-          </ul>
-        </span>
-      </div>
-    );
+    check_third_mun = 'red';
   }
 
   // render app
   return (
     <div>
-      {" "}
+      {' '}
       <Intro post={1} introTodayDate={introTodayDate} />
-      {FirstTweet()}
+      <FirstTweet
+        check_first={check_first}
+        labTests={labTests}
+        summary={summary}
+      />
       <Outro />
       <br />
       <br />
       <Intro post={2} introTodayDate={introTodayDate} />
-      {SecondTweet()}
+      <SecondTweet
+        check_second={check_second}
+        stats={stats}
+        patients={patients}
+      />
```

Comment:
And this, although the names are a bit confusing. I know it's because we send specific tweets with specific content, but do you think there's a way to name these components so the names tell us about the content, not about the purpose? If we decided to switch tweets, then we would need to change component names, and that's silly :)

Reply:
Naming, the hardest thing. Maybe while getting more familiar with the code... I haven't thought much about it while I was concentrating not to break something, which I did anyway :)

Reply:
Sure, one day.

Reply:
Some naming suggestions.


FirstTweet -> **Tests_Active** 
SecondTweet -> **Hospitalized_Deceased**
ThirdTweet -> **Combined** (nothing better at the moment); with extraction of some data into separate file:

**Vaccination_Confirmed_PerAge**:
➡️ Število cepljenih oseb: 19.523 (+3.184). // into new file **Vaccination**
➡️ Skupaj 138.518 potrjenih primerov. //  into new file **Confiremd** 
➡️ Potrjeni primeri po starosti: 0-4 (13), 5-14 (31), 15-24 (198), 25-34 (306), 35-44 (369), 45-54 (331), 55-64 (277), 65-74 (185), 75-84 (95), 85+ (38) **// PerAgeLine -> PerAge **

PatientsLine -> **InHospitals**
➡️ Stanje po bolnišnicah: 

MunicipalitiesLines -> **Cities_SocialFriendly**
➡️ Po krajih: **// facebook friendly
➡️ Po krajih: **// twitter friendly

Or in SCREAMING_SNAKE_CASE.
That's for start. I still have some stupid names in other parts.

Should I go with folder structure like below?

https://github.com/benawad/destiny
Creates a directed graph based on how the JavaScript/TypeScript files import each other
`npx destiny "src/components/**/*.*"`
INFO: Generating tree for: src/components/**/*.*
components
├──Footer.js
├──List.js
├──List\FirstTweet\Percentage.js
├──List\Intro.js
├──List\Legend.js 
├──List\ReorderThirdTweet.js   //temporary file 
├──List\shared\Arrow.js
├──List\shared\DataTranslate.js 
├──List\shared\Delta.js
├──List\shared\InOut.js
├──List\shared\Municipalities.js
├──List\shared\MunicipalitiesLines.js //rename
├──List\shared\PatientsLine.js //rename
├──List\shared\PerAgeLine.js //rename
├──List\shared\Separator.js
├──List\shared\TitleLine.js //rename
├──List\shared\Translate.js
├──List\ThirdTweet.js //rename
├──RandomGenerator.js
├──shared\FirstTweet.js //rename
├──shared\Outro.js
├──shared\SecondTweet.js //renam
└──withListLoading.js

Reply:
@jalezi This seems fine, go for it.

Reply:
@VesterDe maybe we should go for SCREAMING_SNAKE_CASE, so that eslint doesn't go crazy or we don't care?

Reply:
@jalezi Sure, the rest of the project isn't really commited to a linter, so if we manage one here it's a nice start.